tests/bsim/bluetooth/audio/src/bap_stream_tx.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef STREAM_TX_H
+#define STREAM_TX_H
+
+#include <stdint.h>
+
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/audio/bap.h>
+#include <zephyr/bluetooth/audio/cap.h>
+#include <zephyr/types.h>
+
+/**
+ * @brief Initialize TX
+ *
+ * This will initialize TX if not already initialized. This creates and starts a thread that
+ * will attempt to send data on all streams registered with stream_tx_register().
+ */
+void stream_tx_init(void);
+
+/**
+ * @brief Register a stream for TX
+ *
+ * This will add it to the list of streams the TX thread will attempt to send on.
+ *
+ * @param bap_stream The stream to register
+ *
+ * @retval 0 on success
+ * @retval -EINVAL @p bap_stream is NULL
+ * @retval -EINVAL @p bap_stream is not configured for TX
+ * @retval -EINVAL @p bap_stream.codec_cfg contains invalid values
+ * @retval -ENOMEM if not more streams can be registered
+ */
+int stream_tx_register(struct bt_bap_stream *bap_stream);
+
+/**
+ * @brief Unregister a stream for TX
+ *
+ * This will remove it to the list of streams the TX thread will attempt to send on.
+ *
+ * @param bap_stream The stream to unregister
+ *
+ * @retval 0 on success
+ * @retval -EINVAL @p bap_stream is NULL
+ * @retval -EINVAL @p bap_stream is not configured for TX
+ * @retval -EALREADY @p bap_stream is currently not registered
+ */
+int stream_tx_unregister(struct bt_bap_stream *bap_stream);
+
+/**
+ * @brief Test if the provided stream has been configured for TX
+ *
+ * @param bap_stream The stream to test for TX support
+ *
+ * @retval true if it has been configured for TX, and false if not
+ */
+bool stream_tx_can_send(const struct bt_bap_stream *stream);
```

Comment:
Why is it `can_tx`, not `is_tx` or `is_conf`.

Reply:
To follow the `stream_tx` prefit it sounds weird to have `stream_tx_is_send` or `stream_tx_is_conf_send` as `stream_tx` refers to the module, and not the stream